src/qml/FeatureForm.qml

```diff
@@ -277,6 +282,8 @@ Page {
           property var currentFeature: form.model.featureModel.feature
           property var currentLayer: form.model.featureModel.currentLayer
           property bool autoSave: qfieldSettings.autoSave
+          // TODO investigate why StringUtils are not available in ./editorwidget/*.qml files
+          property var stringUtilities: StringUtils
```

Comment:
Can you forward that with the same name or is that not possible?

Reply:
Unfortunately not possible, already tried and got:
qrc:/qml/FeatureForm.qml:286 Property names cannot begin with an upper case letter

Reply:
IMHO the global singleton should also be lowercase (it's a global property and not a class name). What do you think @3nids ?

Reply:
I shamelesly copied the already existing approach. `FeatureUtils`, `FileUtils` and `GeometryUtils` added via `REGISTER_SINGLETON`, therefore I added `StringUtils` the same way. However, the `platformUtils` are lowercase, but they are added via `setContextProperty` as they were already initialized in `qgismobileapp.cpp`.

Reply:
@suricactus I realize that, let's keep it in here for consistency. We can switch that in a followup pull request if @3nids agrees.

Reply:
it makes sense to have it CamelCased.
But why do you need to store it in a property? It's a singleton, so just call it from wherever you need.

Reply:
Sorry, I wasn't completely precise. It's *like* a global property, being a singleton instance and not a class name, hence should IMHO start with a lowercase letter.

Reply:
I'm not sold to this as you're calling static methods but I don't mind so much.
But I would still avoid to store this in a property, no?

Reply:
> I'm not sold to this as you're calling static methods but I don't mind so much.

We are doing that, but we could also call normal methods. I think of it as

```
StringUtils *stringUtils = new StringUtils(); // That's what the REGISTER_SINGLETON  does
stringUtils->doSometing(); // That's what we are doing in here, if doSometing() is static or not does not mather
```


> But I would still avoid to store this in a property, no?

Agreed, `REGISTER_SINGLETON` can stay